homeassistant/components/remote/harmony.py

```diff
@@ -347,6 +349,9 @@ def new_activity(self, activity_info: tuple) -> None:
 
         num_repeats = kwargs.get(ATTR_NUM_REPEATS)
         delay_secs = kwargs.get(ATTR_DELAY_SECS, self._delay_secs)
+        hold_secs = kwargs.get(ATTR_HOLD_SECS)
```

Comment:
Since `ATTR_HOLD_SECS` has a default, use `kwargs[ATTR_HOLD_SECS]`.

(Yes, I know it's done wrong just two lines above. Whoever wrote that should ... oh – it was me ... well, times were different back then and I was young and reckless)

Reply:
LOL, does this mean I'm young and reckless as well? I like that <g>.
Changed.